spectral_cube/io/fits.py

```diff
@@ -44,7 +44,7 @@ def is_fits(input, **kwargs):
         return False
 
 
-def read_data_fits(input, hdu=None, **kwargs):
+def read_data_fits(input, hdu=None, mode='denywrite', **kwargs):
```

Comment:
Isn't mode passed through `kwargs` from `SpectralCube.read` to here? I regularly use `mode='denywrite'` and haven't run into memory issues when dealing with big cubes.